app/controllers/application_controller.rb

```diff
@@ -1,7 +1,18 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :null_session
+  skip_before_action :verify_authenticity_token
```

Comment:
just keep in mind, that this should be removed after we implement auth tokens

Reply:
If we implements JWT then I remove it.